Source/macOS/LoopbackHTTPServer/OIDLoopbackHTTPServer.h

```diff
@@ -72,6 +72,9 @@ typedef enum {
 - (BOOL)start:(NSError **)error;
 - (BOOL)stop;
 
+- (BOOL)hasIPv4Socket;
```

Comment:
Why not something like:

`@property(nonatomic, readonly) NSURL *preferredLoopbackURL;`